### ISSUE TITLE
GLIBC issue with code-tunnel on Rocky8

### ISSUE THREAD

**Issue by ypczuchr** [microsoft/vscode#203520]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: `1.85.2 8b3775030ed1a69b13e4f4c628c612102e30a681 x64`
- OS Version: `Rocky Linux release 8.8 (Green Obsidian)`

Steps to Reproduce:

1. Run command in terminal:  `code serve-web ...`

We are trying to set up VSCode web interface. All works fine on Rocky9, but on Rocky8 we get an error related to GLIBC: 
```
code-tunnel: /lib64/libc.so.6: version `GLIBC_2.29' not found
```
We are aware, than until January 2024 VSCode supports only GLIBC 2.28 and newer, but what about `code-tunnel` binary? Is there a version which can be used with this old GLIBC version? We've checked #192685 issue and install newest insider version, but problem still occurs.

**Comment by deepak1556** [comment]
Can you provide the error from insiders, if your distro supports `GLIBC >= 2.28` it should have worked.

**Comment by ypczuchr** [comment]
My apologies, I've check not insiders version, but stable ones. With insider version all works fine.
Will next stable version be patched as well?

**Comment by deepak1556** [comment]
Thanks for confirming, yes fix will be available in upcoming stable release `1.86`. I will go ahead and close this issue.
